galley/pkg/runtime/projections/serviceentry/convert/convert.go

```diff
@@ -123,22 +121,16 @@ func Endpoints(endpoints *coreV1.Endpoints, pods pod.Cache, nodes node.Cache, ou
 		// Convert the endpoints in this subset.
 		for _, address := range subset.Addresses {
 			locality := ""
+			var labels map[string]string
 
 			ip := address.IP
 			p, hasPod := pods.GetPodByIP(ip)
-			var labels map[string]string
 			if hasPod {
 				labels = p.Labels
+				locality = p.Locality
```

Comment:
Is this changing behavior? Always using pod locality?

Reply:
No change in behavior.  Pods don't have locality, nodes do.  I've merged the node and pod caches into one (podCache). The podCache is now responsible for detecting when pods change (including if their backing node changed). This abstracts away the need to separately track nodes from the converter ... the pod has everything it needs now.